content/events/2019-03-22-general-meeting.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Semi-Annual General Meeting (Food Week FINALE)
+date: 2019-03-14
+start_date: 2019-03-22 17:00:00
+end_date: 2019-03-22 18:30:00
+categories: [Event]
+tags: [Food Week]
+author: Hai Lin Zhang
+location:
+  name: DMP 301
+  address:
+    streetAddress: 6245 Agronomy Road 
+    addressLocality: Vancouver
+    postalCode: V6T 1Z4
+    addressRegion: BC
+    addressCountry: CA
+images:
+  - /files/general-meeting-march2019.svg
+---
+
+![](/files/general-meeting-march2019.svg)
+
+Affect the future of the CSSS and the Cube! Come to our cheap food week **FINALE** and buy bubble tea! See the reveal of the new executive team!
```

Comment:
It would be nice to hyperlink to the food week event, and vice-versa.
```suggestion
Affect the future of the CSSS and the Cube! Come to our [cheap food week](/events/2019/03/19/cube-food-week/) **FINALE** and buy bubble tea! See the reveal of the new executive team!
```